Tests/govuk_ios/govuk_ios_unit_tests/Mocks/MockAnalyticsService.swift

```diff
@@ -0,0 +1,71 @@
+//
+
+import XCTest
+import Logging
+
+final class MockAnalyticsService: AnalyticsService {
```

Comment:
Not sure if/ when this will bite us, but might be worth adding the namespace for AnalyticsService here? i.e `Logging.AnalyticsService`